### PR TITLE
release: 3.26.12

### PR DESCRIPTION
**Pull request by ycjcl868**
## 3.26.12

`2020-02-24`

- 🐞 Fix Input with `readOnly` still clearable by `allowClear`. [#21492](https://github.com/ant-design/ant-design/pull/21492)
- 🐞 Fix Upload won't showing download icon defaultly. [#21496](https://github.com/ant-design/ant-design/pull/21496)
- ⚡️  Improve Button render performance. [#21217](https://github.com/ant-design/ant-design/pull/21217)
-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/release-3.26.12/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/release-3.26.12/CHANGELOG.zh-CN.md)

### CODE REVIEW

**Review by afc163 — APPROVED** [review]

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/commit/e85ec40c9afc335c5b3aef8195b16140aa9b771f

**Review by afc163 — CHANGES_REQUESTED** [review]
https://github.com/ant-design/ant-design/commit/e85ec40c9afc335c5b3aef8195b16140aa9b771f

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ timeline: true
 
 ---
 
+## 3.26.12
+
+`2020-02-24`
+
+- 🐞 Fix Input with `readOnly` still clearable by `allowClear`. [#21492](https://github.com/ant-design/ant-design/pull/21492)
+- 🐞 Fix Upload won't showing download icon defaultly. [#21496](https://github.com/ant-design/ant-design/pull/21496)
+- 💄 Tweak Button render performance. [#21217](https://github.com/ant-design/ant-design/pull/21217)
```
Comment (afc163) [inline CHANGELOG.en-US.md]:
```suggestion
- ⚡️ Improve Button render performance. [#21217](https://github.com/ant-design/ant-design/pull/21217)
```

**Comment by zombieJ** [comment]
Azure 的 CI 在 v3 branch 可以不用管

**Pull request merged**